docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,380 @@
+# which is located at the origin. For this example we are going to work in a 6-dimensional state space our platform will
+# have the following :math:`\vec{x}`.
+#
+# .. math::
+#           \begin{align}\vec{x} = \begin{bmatrix}
+#                                   x\\ \dot{x}\\ y\\ \dot{y}\\ z\\ \dot{z} \end{bmatrix}
+#                                = \begin{bmatrix}
+#                                   0\\ 0\\ 0\\ 0\\ 0\\ 0 \end{bmatrix}\end{align}
+#
+# Because the platform is static we only need to define :math:`(x, y, z)`, any internal interaction with the platform
+# which requires knowledge of platform velocity :math:`(\dot{x}, \dot{y}, \dot{z})` will be returned :math:`(0, 0, 0)`.
+
+# First import the fixed platform
+from stonesoup.platform.base import FixedPlatform
+
+# Define the initial platform position, in this case the origin
+platform_state_vector = StateVector([[0], [0], [0]])
+position_mapping = (0, 1, 2)
+
+# Create the initial state (position, time), notice it is set to the simulation start time defined earlier
+platform_state = State(platform_state_vector, start_time)
+
+# create our fixed platform
+platform = FixedPlatform(states=platform_state,
+                         position_mapping=position_mapping)
+
+# %%
+# We have now created a platform within Stone Soup and located it at the origin of our state space. As previously stated
+# the platform will have a velocity :math:`(\dot{x}, \dot{y}, \dot{z})` of :math:`(0, 0, 0)` which we can check:
+platform.velocity
+
+# %%
+# We can also query the platform orientation:
+platform.orientation
+
+# %%
+# Create a Sensor
+# ---------------
+# Now that we have a platform the next step is to create a sensor which can be added to the platform. In this example a
+# Radar will be created which is capable of measuring the range, bearing and elevation of the target relative to the
+# sensor.
+#
+# The :class:`~.RadarRangeBearingElevation`  provides a sensor wrapper around the
+# :class:`~.CartesianToElevationBearingRange` measurement model. The measurement model provides a time-invariant
+# measurement model, where measurements are assumed to be received in the form of elevation (:math:`\theta`),
+# bearing (:math:`\phi`) and range (:math:`r`) with Gaussian noise in each dimension.
+#
+# The model is described by the following equations:
+#
+# .. math::
+#           \begin{align} \vec{y}_t = h(\vec{x}_t, \vec{v}_t) \end{align}
+#
+# where :math:`\vec{y}_t` is a measurement vector of the form:
+#
+# .. math::
+#           \begin{align} \vec{y}_t = \begin{bmatrix} \theta \\ \phi \\ r \end{bmatrix} \end{align}
+#
+# and :math:`h` is a non-linear model function of the form:
+#
+# .. math::
+#           \begin{align} h(\vec{x}_t,\vec{v}_t) = \begin{bmatrix}
+#                                               asin(\mathcal{z}/\sqrt{\mathcal{x}^2 + \mathcal{y}^2 +\mathcal{z}^2}) \\
```

Comment:
I think you can use \arcsin and \arctan (the latter is the same as atan2 except without solving the quadrant ambiguity - but that point usually gets made in the text because I think atan2 was invented by programmers!)